bee-api/bee-rest-api/CHANGELOG.md

```diff
@@ -21,9 +21,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.2.0 - 2022-XX-XX
 
+### Added
+
+- `ExtendedOutputDto`;
+- `AliasOutputDto`;
+- `FoundryOutputDto`;
+- `NftOutputDto`;
```

Comment:
I think this is missing a lot of stuff

Reply:
Maybe we don't need to be too granular but I'd at least put the new address DTOs, the main FeatureBlock DTO...

Reply:
Added all the new DTO's